src/Model/WineManager.php

```diff
@@ -8,16 +8,31 @@ class WineManager extends AbstractManager
 {
     public const TABLE = 'wine';
 
+    public function selectOneWineById(int $id): array|false
+    {
+        // prepared request
+        $statement = $this->pdo->prepare('SELECT wine.id as id, name, year, price, region, color, grape, partner_id, '
+        . static::TABLE . '.description, p.lastname as lastname, p.firstname as firstname FROM ' . static::TABLE .
+        ' INNER JOIN partner as p ON p.id = wine.partner_id WHERE ' . static::TABLE . '.id=:id');
```

Comment:
instead of putting static::TABLE everywhere, you should put an alias in your query for wine as you do for partner.

Reply:
i admit that's rather ugly